swagger.yaml

```diff
@@ -37,6 +37,33 @@ paths:
           description: "Unauthorised to fetch"
         500:
           $ref: "#/responses/InternalError"
+  /population-types/{population-type}/area-types:
```

Comment:
Doesn't this endpoint exist on dp-cantabular-dimension-api?

Reply:
Yes, but given the area-types belong to a population-type the TL consensus was this was the best place to put it.  I've created a ticket to create some routing from this API to point to dp-cantabular-dimension-api when called - https://trello.com/c/b8Cwk5fa/5672-create-population-types-population-type-area-types-endpoint